pkgs/top-level/all-packages.nix

```diff
@@ -1508,11 +1508,23 @@ in
     cudnn = cudnn_cudatoolkit_10_0;
   };
 
+  libtensorflow-bin_2 = callPackage ../development/libraries/science/math/tensorflow/bin_2.nix {
+    cudaSupport = config.cudaSupport or false;
+    inherit (linuxPackages) nvidia_x11;
+    cudatoolkit = cudatoolkit_10_0;
+    cudnn = cudnn_cudatoolkit_10_0;
+  };
+
   libtensorflow =
     if python.pkgs.tensorflow ? libtensorflow
     then python.pkgs.tensorflow.libtensorflow
     else libtensorflow-bin;
 
+  libtensorflow_2 =
```

Comment:
We already dropped python3.pkgs.tensorflow1 in master. You can also just update libtensorflow to _2?